release/src/main/scripts/verify_release_build.sh

```diff
@@ -103,6 +103,9 @@ else
 fi
 
 echo "======================Starting Clone Repo======================"
+if [[ ! -z `ls | grep ${LOCAL_CLONE_DIR}` ]]; then
+  rm -rf ${LOCAL_CLONE_DIR}
```

Comment:
You can use `[ -d ${LOCAL_CLONE_DIR} ]` (see `man test`); you can change this throughout.